Maven plugin fails with empty example in Schema annotation

The maven plugin execution runs into an error, if the example (string) of a Schema annotation is empty, e.g.
```
@Schema(description = "the name of something", example = "")
private String name;
```

`Execution default of goal io.smallrye:smallrye-open-api-maven-plugin:2.0.14:generate-schema failed: String index out of range: 0`

```
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: 0
    at java.lang.StringLatin1.charAt (StringLatin1.java:47)
    at java.lang.String.charAt (String.java:693)
    at io.smallrye.openapi.runtime.io.JsonUtil.parseValue (JsonUtil.java:271)
    at io.smallrye.openapi.runtime.io.schema.SchemaFactory.lambda$parseSchemaAttr$5 (SchemaFactory.java:292)
    at io.smallrye.openapi.runtime.io.schema.SchemaFactory.readAttr (SchemaFactory.java:320)
    at io.smallrye.openapi.runtime.io.schema.SchemaFactory.parseSchemaAttr (SchemaFactory.java:286)
    at io.smallrye.openapi.runtime.io.schema.SchemaFactory.readSchema (SchemaFactory.java:198)
    at io.smallrye.openapi.runtime.scanner.dataobject.AnnotationTargetProcessor.readSchemaAnnotatedField (AnnotationTargetProcessor.java:223)
    at io.smallrye.openapi.runtime.scanner.dataobject.AnnotationTargetProcessor.processField (AnnotationTargetProcessor.java:155)
    at io.smallrye.openapi.runtime.scanner.dataobject.AnnotationTargetProcessor.process (AnnotationTargetProcessor.java:66)
    at io.smallrye.openapi.runtime.scanner.OpenApiDataObjectScanner.lambda$depthFirstGraphSearch$1 (OpenApiDataObjectScanner.java:261)
```

--- a/core/src/main/java/io/smallrye/openapi/runtime/io/JsonUtil.java
+++ b/core/src/main/java/io/smallrye/openapi/runtime/io/JsonUtil.java
@@ -258,7 +258,7 @@ public final class JsonUtil {
      * @return Extension
      */
     public static Object parseValue(String value) {
-        if (value == null) {
+        if (value == null || value =="") {
             return null;
         }
 

--- a/core/src/main/java/io/smallrye/openapi/runtime/io/JsonUtil.java
+++ b/core/src/main/java/io/smallrye/openapi/runtime/io/JsonUtil.java
@@ -258,7 +258,7 @@ public final class JsonUtil {
      * @return Extension
      */
     public static Object parseValue(String value) {
-        if (value == null || value =="") {
+        if (value == null || value.isEmpty()) {
             return null;
         }
 

